### ISSUE TITLE
Escaping colon character in the format string of date_format validation rule

### ISSUE THREAD

**Issue by gyszalai** [poppinss/indicative#14]
I'd like to validate a string that contains time in 24h syntax 
with a colon between the hours and minutes.

I tried different ways like:

```
time: "date_format:HH:mm"
time: "date_format:'HH:mm'"
time: 'date_format:"HH:mm"'
time: 'date_format:HH\:mm'
time: 'date_format:HH::mm'
```

Neither of them worked. Is it possible somehow?

**Comment by thetutlage** [comment]
Nice catch , fixed as of version 1.2.4

**Comment by gyszalai** [comment]
Thanks!
